pages/index.js

```diff
@@ -1,3 +1,18 @@
+import Head from "next/head";
+import Image from "next/image";
+import SitesList from "../components/SitesList";
+
 export default function Home() {
-  return <h1>Hello Beavers 🦫</h1>;
+  return (
+    <>
+      <Head>
+        <title>Profan</title>
+      </Head>
+      <h1>
+        pro|f<u>a</u>n
```

Comment:
cool that you found this, but mdn recommends not to use this for such a purpose: 
_You should not use <u> to underline text for presentation purposes, or to denote titles of books._

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u?retiredLocale=de
maybe we could use a span and tyole the underline here?

Reply:
Could not find tyole... What does that mean?

Reply:
OH, that went wrong. span was the word I was looking for.